spec/features/trainees/edit_publish_course_details_spec.rb

```diff
@@ -28,12 +28,13 @@
       then_the_section_should_be(not_started)
     end
 
-    scenario "renders an 'in progress' status when details partially provided" do
+    scenario "renders a 'completed' status when details fully provided" do
```

Comment:
Should we add a new test instead of replacing the old one? I think we still want to check that the in progress status is rendered in that situation